crates/starknet-devnet/tests/test_blocks_generation.rs

```diff
@@ -412,6 +418,64 @@ mod blocks_on_demand_tests {
         assert_eq!(get_contract_balance(&devnet, contract_address).await, expected_balance);
     }
 
+    #[tokio::test]
+    async fn blocks_on_interval() {
+        let devnet = BackgroundDevnet::spawn_with_additional_args(&["--block-generation-on", "1"])
+            .await
+            .expect("Could not start Devnet");
+
+        // wait 1 second
+        tokio::time::sleep(time::Duration::from_secs(1)).await;
+
+        let last_block = devnet.get_latest_block_with_tx_hashes().await.unwrap();
+
+        // first is genesis block, second block is generated instantly, third is generated after 1
+        // second
+        assert_eq!(last_block.block_number, 2);
+    }
+
+    #[tokio::test]
+    async fn blocks_on_interval_dump_and_load() {
+        let mode = "exit";
+        let dump_file = UniqueAutoDeletableFile::new(
+            ("blocks_on_interval_dump_and_load_".to_owned() + mode).as_str(),
```

Comment:
Why this complicated string generation? It can be anything really, a unique suffix is appended to the name anyway.

Reply:
I added it for readability I also comment remove in:
```
impl Drop for UniqueAutoDeletableFile {
    fn drop(&mut self) {
        remove_file(&self.path)
    }
}
```

and I check all files generated so it's also easier to read and check them

Reply:
IMO it just reduces the clarity by adding an second or two to understanding the command. Something like this would be enough, but we don't have to spend time on such trivial matter: `let dump_file = UniqueAutoDeletableFile::new("interval_dump");`

Reply:
Fixed in 54396d5f718fff386541c8d812e7943420228b16